packages/block-editor/src/components/link-control/style.scss

```diff
@@ -209,7 +216,8 @@ $preview-image-height: 140px;
 
 	.block-editor-link-control__search-item-title {
 		display: block;
-		margin-bottom: 0.2em;
+		margin-top: 0.2em;
+		font-size: 1.1em;
```

Comment:
In general we don't use em for font sizes or lineheights unless in edgecases. I'm aware this was existing code and you're moving it around, but since you're in here it might be nice to update. 1.1em here computes to 14.3px font size and the line height and margins aren't that round either:

<img width="379" alt="Screenshot 2021-06-24 at 15 06 35" src="https://user-images.githubusercontent.com/1204802/123267909-cf9e3b00-d4fd-11eb-8479-b89306ab9c79.png">

If I remove the top margin, change the font size to `$default-font-size`, and set the line-height of the parent container to `1.5` (unitless), I get this:

<img width="545" alt="Screenshot 2021-06-24 at 15 08 31" src="https://user-images.githubusercontent.com/1204802/123268228-2572e300-d4fe-11eb-8976-45eae2f820f6.png">

Happy to push these changes, by the way, just let me know.